sunpy/map/sources/soho.py

```diff
@@ -153,6 +153,32 @@ def is_datasource_for(cls, data, header, **kwargs):
         """Determines if header corresponds to an LASCO image."""
         return header.get('instrume') == 'LASCO'
 
+    @property
+    def mask_occulter(self):
```

Comment:
There is a standing policy that we do not add source-specific methods to map, or put another way "all maps for all sources have exactly the same API". Therefore I would like to see this functionality moves somewhere else.

Given this seems very LASCO specific (and involves downloading a data file) I think the best place for this to live is in sunkit-instruments.